loader/vk_loader_platform.h

```diff
@@ -288,8 +288,10 @@ static inline char *loader_platform_executable_path(char *buffer, size_t size) {
 #include <libproc.h>
 static inline char *loader_platform_executable_path(char *buffer, size_t size) {
     pid_t pid = getpid();
-    int ret = proc_pidpath(pid, buffer, size);
-    if (ret <= 0) return NULL;
+    int ret = proc_pidpath(pid, buffer, (uint32_t)size);
```

Comment:
> Unfortunately, this would make the code inconsistent with the other platforms. Every other platform takes size_t, and having one platform differ would mean that loader_platform_executable_path performs narrowing that wouldn't be obvious.

Is this better? I mean the narrowing conversion has to happen. I can add an error check if size is greater than uint32_t max